### ISSUE TITLE
auth error: response code is not 200: 401

### ISSUE THREAD

**Issue by march5350** [dakhnod/WatchAuthenticator#4]
When I try to get the shared secret form my Fossil watch. I get the error _auth error: response code is not 200: 401_. The links I used where the ones in the readme, namely
- https://api.skagen.linkplatforms.com/v2.1/rpc/auth/
- https://api.skagen.linkplatforms.com/v2.1/rpc/device/
- https://api.skagen.linkplatforms.com/v2/users/me/devices
- https://api.skagen.linkplatforms.com/v2/users/me/devices/%s/secret-key
After that I use my credentials to login. I created my Fossil account using the Google login method. Could this maybe the the reason of the issue?

**Comment by dakhnod** [comment]
So you entered your Google credentials?

**Comment by march5350** [comment]
Yep, I did. I also checked again if these where the good credentials by logging out of the Fossil app and logging back in again. And I forgot to mention that I tried the links with https://c.fossil.com/ as well as https://api.skagen.linkplatforms.com.

**Comment by dakhnod** [comment]
You need to use fossil native credentials. I was planning on writing that down somewhere...

**Comment by march5350** [comment]
That explains a lot. Thanks for looking into it!
